_pages/fr/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: index
+title:  "Activate en Mozilla"
```

Comment:
I couldn't find the 'en? you are talking about. If it's the ttilte, has to be translate it